04-account-kit-manage-permissions/index.ts

```diff
@@ -0,0 +1,48 @@
+// SDK exposes all you need
+import { Chains, PublicKey, SigningRequest, Permission } from '@[SOMETHING]kit'
+import { Account } from '@[SOMETHING]kit/account'
+
+async function main() {
+    // Loads account
+    const account: Account = await Account.load("teamgreymass", Chains.EOS)
+
+    // Manage account Permissions
+
+    // Get a permission
+    const permission: Permission = account.permissions.get("active")
+
+    // Get its public keys
+    const publicKeys: PublicKey[] = permission.keys
+
+    // Get its accounts
+    const accounts: Account[] = permission.accounts
+
+    // Add a new permission
+    const addPermissionEsr: SigningRequest = account.permissions.add('active', PublicKey.from('PUB_K1_5rcqJ8eNgNXooYyz9PTX2a2zZBktWaWezhXYGN2hrXWqfHSWYY'))
```

Comment:
Just like with the comment about `update`, the `add` should probably take in an entire permission object that contains accounts/keys/waits.

Reply:
Will also need to account for `parent` somehow.

Reply:
Reference: https://bloks.io/account/eosio?loadContract=true&tab=Actions&account=eosio&scope=eosio&limit=100&action=updateauth